daemon/container.go

```diff
@@ -204,12 +204,12 @@ func (container *Container) LogEvent(action string) {
 	}
 }
 
-func (container *Container) getResourcePath(path string) (string, error) {
+func (container *Container) GetResourcePath(path string) (string, error) {
```

Comment:
If these are exported, they should have godoc comment explaining exactly what is provided.